easybuild/easyconfigs/l/LAST/LAST-869-intel-2017a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'MakeCp'
+
+name = 'LAST'
+version = '869'
+
+homepage = 'http://last.cbrc.jp/'
+description = """ LAST finds similar regions between sequences. The main technical innovation
+ is that LAST finds initial matches based on their multiplicity,
+ instead of using a fixed length (e.g. BLAST uses 11-mers)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'cstd': 'c++11', 'opt': True}
```

Comment:
@hajgato Why the `'opt': True` (which enables using `-O3` rather than `-O2`), any particular reason for this?

Reply:
@boegel: taken form `last-869/makefile`, line 1:
```
CXXFLAGS = -O3 -std=c++11 -pthread -DHAS_CXX_THREADS
```

Reply:
OK so you're trying to mimic this behaviour, I see.

In general, `-O3` is less "safe" than `-O2` though (in terms of precision, etc.).

And the benefit is usually fairly small.